intel-sgx/ias/src/api.rs

```diff
@@ -230,46 +204,46 @@ impl SafeToDeserializeInto for Unverified {}
 #[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
 #[serde(bound(deserialize = "V: SafeToDeserializeInto"))]
 #[serde(rename_all = "camelCase")]
-pub struct VerifyAttestationEvidenceResponse<V: VerificationType = Verified> {
-    pub(crate) id: String, // TODO: decimal big integer
+pub struct VerifyAttestationEvidenceResponse<V: VerificationType = VerifiedSig> {
+    id: String, // TODO: decimal big integer
 
-    pub(crate) timestamp: String, // TODO: DateTime<UTC>, but DateTime serde doesn't
-                                  // like the IAS timestamp format.
+    timestamp: String, // TODO: DateTime<UTC>, but DateTime serde doesn't
+                           // like the IAS timestamp format.
 
     #[serde(default = "two", skip_serializing_if = "less_than_three")]
-    pub(crate) version: u64,
+    version: u64,
 
-    pub(crate) isv_enclave_quote_status: QuoteStatus,
+    isv_enclave_quote_status: QuoteStatus,
 
     #[serde(with = "serde_bytes")]
-    pub(crate) isv_enclave_quote_body: Vec<u8>,
+    isv_enclave_quote_body: Vec<u8>,
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub(crate) revocation_reason: Option<u32>, // TODO: enum RFC5280 CRLReason
+    revocation_reason: Option<u32>, // TODO: enum RFC5280 CRLReason
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub(crate) pse_manifest_status: Option<String>, // TODO: enum per IAS spec
+    pse_manifest_status: Option<String>, // TODO: enum per IAS spec
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub(crate) pse_manifest_hash: Option<String>, // TODO: base16 blob
+    pse_manifest_hash: Option<String>, // TODO: base16 blob
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub(crate) platform_info_blob: Option<String>, // TODO: base16 blob
+    platform_info_blob: Option<String>, // TODO: base16 blob
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub(crate) nonce: Option<String>,
+    nonce: Option<String>,
 
     #[serde(skip_serializing_if = "Option::is_none", with = "self::serde_option_bytes")]
-    pub(crate) epid_pseudonym: Option<Vec<u8>>, // base64
+    epid_pseudonym: Option<Vec<u8>>,
 
     #[serde(skip_serializing_if = "Option::is_none", rename = "advisoryURL")]
-    pub(crate) advisory_url: Option<String>,
+    advisory_url: Option<String>,
 
     #[serde(skip_serializing_if = "Option::is_none", rename = "advisoryIDs")]
-    pub(crate) advisory_ids: Option<Vec<IasAdvisoryId>>,
+    advisory_ids: Option<Vec<IasAdvisoryId>>,
 
     #[serde(skip)]
-    pub(crate) type_: PhantomData<V>
+    type_: PhantomData<V>
 }
 
 impl<V: VerificationType> VerifyAttestationEvidenceResponse<V> {
```

Comment:
Why are these methods available unverified? Is this for the IasVerificationResult? Then the unverified version should be `pub(crate)`.

Reply:
Yes that's solely for the `IasVerificationResult`. Fixed